### PR TITLE
[TASK] Prepare release of version 7.0.0

### PR DESCRIPTION
**Pull request by oliverklee**
We should have version 7.0.0 with PHP 8.2 support out of the door before the final PHP 8.2 will get released later this month.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,8 +5,21 @@ This project adheres to [Semantic Versioning](https://semver.org/).
...
+
+### Removed
+
+### Fixed
+
+## 7.0.0
+
 ### Added
 - Add support for PHP 8.2 (#1155)
+- Add official support for Symfony 6
```
Comment (JakeQZ) [inline CHANGELOG.md:22]: This was added in #1065 which was included in 6.0.0 so I don't think this changelog entry belongs here.

**Review by JakeQZ — CHANGES_REQUESTED** [review]
I think the changelog entry about Symfony 6 doesn't belong.

File: CHANGELOG.md (hunk 1 above)
Comment (oliverklee) [reply CHANGELOG.md:22]: Oh, right, I think we missed that. Going to change that back in a second.

**Review by JakeQZ — APPROVED** [review]
Looks fine now.

**Pull request merged**